Core/Command.cs

```diff
@@ -2274,17 +2274,20 @@ public string ParseCommand(string sText, bool bSendToGame = false, bool bUserInp
                                                         {
                                                             try
                                                             {
-                                                                if (oArgs.Count < 5) throw new Exception("Syntax Error: Window Name, Width, and Height are required.");
+                                                                if (oArgs.Count < 4) throw new Exception("Syntax Error: Window Name, and one of (Width, Height, Top, or Left are required.");
```

Comment:
This needs a closing peren after "left" I think

Reply:
I added the peren and reverted the swapped count checks

Reply:
Yes it does.